packages/react/src/date-picker/reactDayPickerStyles.ts

```diff
@@ -112,18 +115,21 @@ export const reactDayPickerStyles = {
 		'&:last-of-type': { marginRight: 0 },
 	},
 	'.rdp-table': {
-		margin: 0,
-		maxWidth: `calc(${cellSize} * 7)`,
 		borderCollapse: 'collapse',
+		height: cellSizeSmall,
```

Comment:
Why would the table be given the height of a cell?

Reply:
Well, that's awkward. Not sure why I did that!

- [ ] Restore `.rdp-table` properties

Reply:
I'm not sure any of these table height/width styles actually matter